### ISSUE TITLE
Feature requests

### ISSUE THREAD

**Issue by vzorglub** [sandro-k/MQTTLensChromeApp#11]
Hello,
Could we have the option to have a 24 hour time stamp, please?
Also, how do we clear the log without having to close the app and restart?
Thanks

**Comment by sandro-k** [comment]
Yes 24 hour stamp would be nice. Currently clearing is not possible without removing the subscription.
